package.json

```diff
@@ -33,7 +34,7 @@
     "msp"
   ],
   "license": "MIT",
-  "main": "index.js",
```

Comment:
Yeah for this to yarn link it for testing, it complains that the main entry is wrong without adding the dist folder to the path

Reply:
Yeah, that's true. But if we publish like this, `yarn install egnyte-resellers` will fail with something like `main file not found: dist/index.js` because there will not be a `dist` directory when published. The published package contains only the contents of `dist` in the root directory. So for now I think we'll just have to remember to change this to `dist/index.js` when testing with `yarn link` and then change it back when opening a PR and/or publishing.